app/components/Frame.js

```diff
@@ -46,7 +46,7 @@ const mapStateToProps = (state => {
   //should we clean/shorten the initial state structures?
 
   return {
-      //allproducts : state.products.allproducts,
+      allproducts : state.products.allProducts,
```

Comment:
watch allproducts  and allProducts... (w preference for the later)... there are a few allproducts left in the code